lib/data/repositories/collection_repository.dart

```diff
@@ -90,6 +92,8 @@ class CollectionRepositoryImpl implements CollectionRepository {
               CollectionKeys.contributors: collection.contributors.map(_contributorSerializer.to),
               CollectionKeys.uniqueMemosAmount: collection.uniqueMemosAmount,
               CollectionKeys.uniqueMemoExecutionsAmount: collection.uniqueMemoExecutionsAmount,
+              CollectionKeys.isPremium: collection.isPremium,
+              CollectionKeys.productInfo: _productInfoSerializer.to(collection.productInfo),
```

Comment:
Ai esse `productInfo` não existindo caso n seja premium haveria esse if aqui:

```suggestion
              if(collection.isPremium)
              CollectionKeys.productInfo: _productInfoSerializer.to(collection.productInfo),
```